### PR TITLE
Refresh translation files

### PR DESCRIPTION
**Pull request by rnicoll**
Refresh the most active translation files. As we're not set up with the same tooling as Bitcoin, we get slightly different results compared to the files they generate, so I've picked only specific files to update to reduce churn. If we have translation volunteers for other languages we can update those languages at the time.

* Chinese
* French
* German
* Indonesian
* Italian
* Japanese
* Korean
* Netherlands
* Spanish
* Russian
* Turkish

### CODE REVIEW

**Comment by AbcSxyZ** [comment]
Shared on a french discord to see if someone want to help. May I suggest to tweet about to facilitate this kind of work ? Can be relayed in communities of each language, and it is an accessible task to which to contribute.

May be done more quicly that way, and in any case I gess finding people from each language will be tricky without communication.

**Comment by Shira518** [comment]
Can help you with the french translation if you want

**Comment by AbcSxyZ** [comment]
@Shira518 The goal of this pull request is to find some translators, so your help is welcome :)

You can do your PR on rnicoll repo (I guess, correct me @rnicoll otherwise), try to share your progress/work here to avoid duplicate work !

(Edit your comment with checkbox @rnicoll ?)

**Comment by AbcSxyZ** [comment]
@Shira518 let me know if you need help to do your PR, I'm seeing you don't have any activity on GitHub, maybe you're not comfortable with this.

## Step follow to translate and do PR to rnicoll

Briefly, to PR on @rnicoll repo, if anyone need it.
1./ First, fork [dogecoin/dogecoin](https://github.com/dogecoin/dogecoin).
```
#clone your own fork
git clone git@github.com:your-username/dogecoin.git

# Get rnicoll repository
git remote add rnicoll https://github.com/rnicoll/dogecoin
git pull rnicoll

# Go to rnicoll branch for this pr, will get you all files you need in the good state
git checkout 1.14.5-dev-strings-all

# Finally, create your own branch to do your work
git checkout -b 1.14.5-dev-translation-my-langage
```

Now you can work on files and translates.

When you finished, to send your work.
```
# Push changes to your fork
git push origin 1.14.5-dev-translatoin-my-langage
```

Go to github, on [rnicoll repository](https://github.com/rnicoll/dogecoin). You should see a new yellow windows at the top saying "[your branch] had recent pushes x minutes ago".
Click on compare & pull request.

Make sure you send from the right branch to the right branch. By default Github will try to push your changes on master branch of rnicoll repo, change it here to the branch you started to work with, `1.14.5-dev-strings-all`. On the list of commit you will try to PR, it should show only yours, otherwise there is a problem.

Hope this help.

**Comment by dogespacewizard** [comment]
ive added some korean translations as a pr to rnicolls repo per the above instructions, wasnt able to do all of them and would love a real programmer who is fluent in korean to mayhaps double check what I've added
https://github.com/rnicoll/dogecoin/pull/29

**Comment by patricklodder** [comment]
@rnicoll did you mean to make the new translations blocking to this PR? Or otherwise want that? If yes, then I will wait with review until you have more translations in (may make it harder to merge though)

**Comment by magneticat** [comment]
Do you need help with the Italian translation?

**Comment by AbcSxyZ** [comment]
@magneticat The language is listed, and it seems no one has declared themselves to do so, I guess yes :)

**Comment by rnicoll** [comment]
Wow - I had meant to get these in as a baseline and then apply translations to them later. I am inclined to sanity check and merge in the PRs which have been raised, and then merge this, and any further ones we raise per-language. That seem reasonable @patricklodder @AbcSxyZ ?

**Comment by patricklodder** [comment]
> Wow - I had meant to get these in as a baseline and then apply translations to them later.

Yeah that's what I thought.

> I am inclined to sanity check and merge in the PRs which have been raised

Fine, but please don't squash them though, as that would set me back to scratch in review!

**Comment by rnicoll** [comment]
> 
> 
> > Wow - I had meant to get these in as a baseline and then apply translations to them later.
> 
> Yeah that's what I thought.
> 
> > I am inclined to sanity check and merge in the PRs which have been raised
> 
> Fine, but please don't squash them though, as that would set me back to scratch in review!

Sorry, I should have said I figured you'd just run the `lupdate` command and diff the results. The command used to update _all_ of them is:

```
QT_SELECT=qt5 /usr/lib/x86_64-linux-gnu/qt5/bin/lupdate qt/bitcoinstrings.cpp qt/forms/addressbookpage.ui qt/forms/askpassphrasedialog.ui qt/forms/coincontroldialog.ui qt/forms/editaddressdialog.ui qt/forms/helpmessagedialog.ui qt/forms/intro.ui qt/forms/importkeysdialog.ui qt/forms/modaloverlay.ui qt/forms/openuridialog.ui qt/forms/optionsdialog.ui qt/forms/overviewpage.ui qt/forms/paperwalletdialog.ui qt/forms/receivecoinsdialog.ui qt/forms/receiverequestdialog.ui qt/forms/debugwindow.ui qt/forms/addpeerdialog.ui qt/forms/testpeerdialog.ui qt/forms/sendcoinsdialog.ui qt/forms/sendcoinsentry.ui qt/forms/signverifymessagedialog.ui qt/forms/transactiondescdialog.ui qt/bantablemodel.cpp qt/bitcoinaddressvalidator.cpp qt/bitcoinamountfield.cpp qt/bitcoingui.cpp qt/bitcoinunits.cpp qt/clientmodel.cpp qt/csvmodelwriter.cpp qt/guiutil.cpp qt/intro.cpp qt/modaloverlay.cpp qt/networkstyle.cpp qt/notificator.cpp qt/optionsdialog.cpp qt/optionsmodel.cpp qt/peertablemodel.cpp qt/platformstyle.cpp qt/qvalidatedlineedit.cpp qt/qvaluecombobox.cpp qt/rpcconsole.cpp qt/splashscreen.cpp qt/trafficgraphwidget.cpp qt/utilitydialog.cpp qt/peerdialog.cpp qt/verticallabel.cpp qt/bitcoin.cpp qt/winshutdownmonitor.cpp qt/addressbookpage.cpp qt/addresstablemodel.cpp qt/askpassphrasedialog.cpp qt/coincontroldialog.cpp qt/coincontroltreewidget.cpp qt/editaddressdialog.cpp qt/importkeysdialog.cpp qt/openuridialog.cpp qt/overviewpage.cpp qt/paymentrequestplus.cpp qt/paymentserver.cpp qt/receivecoinsdialog.cpp qt/receiverequestdialog.cpp qt/recentrequeststablemodel.cpp qt/sendcoinsdialog.cpp qt/sendcoinsentry.cpp qt/signverifymessagedialog.cpp qt/transactiondesc.cpp qt/transactiondescdialog.cpp qt/transactionfilterproxy.cpp qt/transactionrecord.cpp qt/transactiontablemodel.cpp qt/transactionview.cpp qt/walletframe.cpp qt/walletmodel.cpp qt/walletmodeltransaction.cpp qt/walletview.cpp qt/addressbookpage.h qt/addresstablemodel.h qt/askpassphrasedialog.h qt/bantablemodel.h qt/bitcoinaddressvalidator.h qt/bitcoinamountfield.h qt/bitcoingui.h qt/bitcoinunits.h qt/clientmodel.h qt/coincontroldialog.h qt/coincontroltreewidget.h qt/csvmodelwriter.h qt/editaddressdialog.h qt/guiconstants.h qt/guiutil.h qt/intro.h qt/importkeysdialog.h qt/macdockiconhandler.h qt/macnotificationhandler.h qt/modaloverlay.h qt/networkstyle.h qt/notificator.h qt/openuridialog.h qt/optionsdialog.h qt/optionsmodel.h qt/overviewpage.h qt/paymentrequestplus.h qt/paymentserver.h qt/peertablemodel.h qt/platformstyle.h qt/qvalidatedlineedit.h qt/qvaluecombobox.h qt/receivecoinsdialog.h qt/receiverequestdialog.h qt/recentrequeststablemodel.h qt/rpcconsole.h qt/sendcoinsdialog.h qt/sendcoinsentry.h qt/signverifymessagedialog.h qt/splashscreen.h qt/trafficgraphwidget.h qt/transactiondesc.h qt/transactiondescdialog.h qt/transactionfilterproxy.h qt/transactionrecord.h qt/transactiontablemodel.h qt/transactionview.h qt/utilitydialog.h qt/peerdialog.h qt/verticallabel.h qt/walletframe.h qt/walletmodel.h qt/walletmodeltransaction.h qt/walletview.h qt/winshutdownmonitor.h qt/macdockiconhandler.mm qt/macnotificationhandler.mm -locations none -no-obsolete -ts qt/locale/*.ts
```

You may want to revert the ones not touched (in particular note that English includes locations, while no other files have locations), but that should be faster than manually reviewing each file.

**Comment by patricklodder** [comment]
Yeah that's about what I did, on your previous commit. So now I just have to review any commits you add ❤️

**Comment by AbcSxyZ** [comment]
Yeah, because I suggested to PR to your repo I agree, would be great if you can merge it :)

If you can figure out a way to signal progress for contributors, I didn't go deep into details but seem to have substantial work, maybe contributors can help each others.
Using checkboxes and linking to WIP PR in your first comment may help.

**Comment by patricklodder** [comment]
> Using checkboxes and linking to WIP PR in your first comment may help.

The point was that we'd merge this with just Ross' update and then contrive could just open their PR independently. But all good, we got this.

**Review by patricklodder — COMMENTED** [review]
The only odd thing I spotted was that some existing translations seem to have moved, but have not retained their translations, e.g. "Such sending addresses". So by not retaining the "outdated" translations, some prior work seems to get lost.

Not sure if that is worth fixing now, or dealing with it during translation contributions.

**Comment by rnicoll** [comment]
"Such sending addresses" is a weird one, because it shows up in two closely related variations:

* "Such sending addresses" on addressbookpage.cpp line 62
* "Such sending addresses..." on bitcoingui.cpp line 388

Qt treats them as separate, and I think we've historically had just one actually translated (and not in all files). They should always appear somewhere in the new file though, if they were in the old one, let me know if I've missed any cases where they're genuinely lost.

File: src/qt/locale/bitcoin_de.ts [hunk 1]
```diff
@@ -305,14 +342,6 @@
         <source>&amp;Change Passphrase...</source>
         <translation>Passphrase &amp;ändern...</translation>
     </message>
-    <message>
-        <source>Such &amp;sending addresses...</source>
-        <translation>&amp;Zahlungsadressen...</translation>
-    </message>
```
Comment (patricklodder) [inline src/qt/locale/bitcoin_de.ts:311]: Original translation that gets `-`

File: src/qt/locale/bitcoin_de.ts [hunk 2]
```diff
@@ -551,12 +586,47 @@
...
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <source>Print paper wallets</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <source>&amp;Such sending addresses...</source>
+        <translation type="unfinished"></translation>
+    </message>
```
Comment (patricklodder) [inline src/qt/locale/bitcoin_de.ts:600]: New translation entry, without the translation

**Review by patricklodder — COMMENTED** [review]
I tagged the former and latter translations. But I think I know what causes this: the `&amp;` to signify the keyboard shortcut has changed position - that would make sense.

**Review by patricklodder — APPROVED** [review]
ACK, with the note that some of the translations not being fully retained is suboptimal, especially on menu entries, but I don't see a better way to do this right now without taking a month, so this will have to do.

**Pull request merged**